diff --git a/pyproject.toml b/pyproject.toml
index f46b5f8..b1ec7ce 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,7 @@ build-backend = "setuptools.build_meta"
 
 # Project settings -----------------------------------------------------------------------------------------------------
 [project]
-name = "ultralytics"
+name = "ultralytics-non-opencv"
 dynamic = ["version"]
 description = "Ultralytics YOLO 🚀 for SOTA object detection, multi-object tracking, instance segmentation, pose estimation and image classification."
 readme = "README.md"
@@ -63,7 +63,6 @@ classifiers = [
 dependencies = [
     "numpy>=1.23.0",
     "matplotlib>=3.3.0",
-    "opencv-python>=4.6.0",
     "pillow>=7.1.2",
     "pyyaml>=5.3.1",
     "requests>=2.23.0",
